tasks/main.yml

```diff
@@ -17,6 +17,19 @@
     query: "message.* | [?label==`{{ item.label }}`]._id"
   register: check_in_place
 
+- name: Disable/Enable a check
+  delegate_to: localhost
+  nodeping:
+    action: disable
+    target: "{{ item.0.target }}"
+    enabled: "{{ nodeping_checks_enabled|bool }}"
+    token: "{{ nodeping_api_token }}"
+  when:
+    - item.1.ansible_facts.result | length > 0
+  with_together:
+    - "{{ nodeping_checks }}"
+    - "{{ check_in_place.results }}"
```

Comment:
- item.1.ansible_facts.result | length > 0
will prevent enabling/disabling a check before it got created. It's very important that the enabling/disabling part comes before the creation. Otherwise a disabled check can never get enabled again. (see: https://nodeping.com/docs-api-checks.html#disableall)